File: cpp/problems/syntax/syntax010.cpp
#include <iostream>
using namespace std;

int main(int argc, char *argv[]) {
    int n = 12321;
    int *num, *inv;
    num = &n;

    cout << *num << endl;
    n++;
    cout << *num << endl;
    return 0;
}

File: objective-c/syntax/syntax010.m
#import <Foundation/Foundation.h>

// class declaration
@interface SimpleClass : NSObject
- (void) PrintCommand;
@end

// class implementation
@implementation SimpleClass
- (void) PrintCommand {
    NSLog(@"printing command...");
}
@end

int main (int argc, const char * argv[]) { 
    NSLog (@"running...");
    // declaring and allocating an instance of the class
    SimpleClass *simpleClass = [[SimpleClass alloc] init];
    // calling method of class instance
    [simpleClass PrintCommand];
    return 0;
}
